packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership-form.jsx

```diff
@@ -94,8 +119,22 @@ const ProofOfOwnershipForm = ({ cards, updateAccountStatus }) => {
                         files_to_send.documentType = 'proof_of_ownership';
                         const response = await uploader.upload(files_to_send);
                         if (response.warning) {
-                            // eslint-disable-next-line no-console
-                            console.warn(response);
+                            if (response.warning.trim() === 'DuplicateUpload') {
```

Comment:
please add optional chaining here to prevent errors

Reply:
I'll add this one